File: C_Programming/Practise_Programs/PP_140.c
//////////////////////////////////////////////////////////////////
//
//  File Name         : PP_140
//
//  Description       : This program demonstrates the display of an
//                      integer value in Decimal, Octal and Hexadecimal
//                      number systems using format specifiers.
//
//  Author            : Shubham Somanath Gadhe
//  Date              : 17/08/2026
//
//  Time Complexity   : O(1)
//  Space Complexity  : O(1)
//
//////////////////////////////////////////////////////////////////

#include<stdio.h>

int main()
{
    int No = 197;

    printf("Decimal : %d\n", No);
    printf("Octal : %o\n", No);
    printf("HexaDecimal : %x\n", No);
    
    return 0;
}

//////////////////////////////////////////////////////////////////
//
//  Sample Input :
//      No input required.
//
//  Sample Output :
//
//      Decimal : 197
//      Octal : 305
//      HexaDecimal : c5
//
//////////////////////////////////////////////////////////////////

File: Java_Programming/Practise_Programs/PP_140.java
//////////////////////////////////////////////////////////////////
//
//  File Name         : PP_140
//
//  Description       : This program prints a decreasing star
//                      pattern using nested for loops. The number
//                      of stars decreases by one in each row.
//
//  Author            : Shubham Somanath Gadhe
//  Date              : 31/08/2026
//
//  Time Complexity   : O(n^2)
//  Space Complexity  : O(1)
//
//////////////////////////////////////////////////////////////////

class PP_140
{
    public static void main(String args[])
    {
        for(int i = 1; i <= 5; i++)
        {
            for(int j = 1; j <= 6 - i; j++)
            {
                System.out.print("* ");
            }

            System.out.println();
        }
    }
}

//////////////////////////////////////////////////////////////////
//
//  Sample Input :
//      No input required.
//
//  Sample Output :
//
//      * * * * *
//      * * * *
//      * * *
//      * *
//      *
//
//////////////////////////////////////////////////////////////////